### ISSUE TITLE
Massive FPS Drop with Puck Lights

### ISSUE THREAD

**Issue by Zsynh** [Jimk72/Icarus_Mods#26]
Heyo -

Trying out IMM and a few mods for the first time, neat system - thanks! I put up about 8 puck lights in my base, and when I switched them on, my FPS went from 35 to 15. Turning them off at the switch rebounded back to 26. Is this a known issue?

I've got a beefy rig running Epic settings without issue otherwise. Thanks!

**Comment by Jimk72** [comment]
Multiple puck lights will drop your fps and 10-15fps. The more you add the more the hit to fps. It all depends on your card, memory and cpu. Sent from my iPhoneOn Jul 12, 2024, at 2:08 PM, airynd ***@***.***> wrote:﻿
Heyo -
Trying out IMM and a few mods for the first time, neat system - thanks! I put up about 8 puck lights in my base, and when I switched them on, my FPS went from 35 to 15. Turning them off at the switch rebounded back to 26. Is this a known issue?
I've got a beefy rig running Epic settings without issue otherwise. Thanks!
